portfolio/src/main/webapp/script.js

```diff
@@ -55,8 +55,147 @@ function deleteTask(task) {
 }
 
 /** Creates a map and adds it to the page. */
+// function createMap() {
+//   const map = new google.maps.Map(
+//       document.getElementById('map'),
+//       {center: {lat: 37.422, lng: -122.084}, zoom: 16});
+// }
+
 function createMap() {
-  const map = new google.maps.Map(
-      document.getElementById('map'),
-      {center: {lat: 37.422, lng: -122.084}, zoom: 16});
-}
+
+        // Create a new StyledMapType object, passing it an array of styles,
```

Comment:
It looks like the formatting in this function is off. I think if you bring everything in this function to the left by 4 spaces, you should be fine.

Reply:
Hi! I fixed this in my new pull request info window :) . I was trying to fix this and push it to this branch but it was not letting me. Idk why